_data/codebase/v2_3/open-source/bin-magento.yml

```diff
@@ -5107,7 +5107,11 @@ commands:
 
         <comment>bin/magento queue:consumers:start someConsumer --area-code='adminhtml'</comment>
 
-    To save PID enter path:
+    To do not run multiple copies of one consumer simultaneously:
+
+        <comment>%command.full_name% someConsumer --single-thread'</comment>
+
+    To save PID enter path (This option is deprecated, use --single-thread instead):
 
         <comment>bin/magento queue:consumers:start someConsumer --pid-file-path='/var/someConsumer.pid'</comment>
```

Comment:
```suggestion
```

Reply:
You can remove the PID task description and example because you indicate the --pid-file-path option is deprecated in the option descriptions.